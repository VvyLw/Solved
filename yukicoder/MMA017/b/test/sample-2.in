10 5
227930076 836334727 108597970 656892815 455743732 901045388 302006162 256603330 228958951 169578258
