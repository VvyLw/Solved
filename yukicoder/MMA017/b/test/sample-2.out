6231622091586614
